experiment/stacks/simulation/js/stack-exercise.js

```diff
@@ -31,6 +31,14 @@ function clearCanvas(ctx) {
 }
 function drawStackStructure(ctx) {
 ctx.beginPath();
+txta = "Stack A";
+txtb = "Stack B";
+stackA_canvas.fillStyle = "black";
+stackA_canvas.font = "20px arial";
+stackB_canvas.font = "20px arial";
+stackB_canvas.fillStyle = "black";
+stackA_canvas.fillText(txta,"220","270"); 
+stackB_canvas.fillText(txtb,"220","270"); 
```

Comment:
Why would you place both texts at the same location? This code doesn't look correct to me, have you tested it?

Reply:
Since each stack is a canvas of a certain width and height, i would place the labels in the same location with respect to the width and height of the canvas. I have tested the code and here is a picture of the result.
<img width="1216" alt="Screen Shot 2022-06-08 at 10 32 10 AM" src="https://user-images.githubusercontent.com/106731226/172535685-eef04258-43bb-4e43-8c47-6b9e41d97800.png">

Reply:
I understand, didn't notice that each stack has it's own canvas.
Can you add a little gap between the text and the stack? Currently the text looks too close to the stack.

Reply:
Yes for sure

Reply:
I have made the changes. Please let me know if I need to make any more. Thanks